keyboards/mechlovin/adelais/standard_led/rules.mk

```diff
@@ -1,3 +1,4 @@
 BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
 BACKLIGHT_DRIVER = pwm
-RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
\ No newline at end of file
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+ENCODER_ENABLE = yes
```

Comment:
Wouldn't it make more sense to move this to `keyboards/mechlovin/adelais/standard_led/rev3/rules.mk`?  
Therefore the feature is not enabled for everybody.